tests/unit/splinterdb_stress_test.c

```diff
@@ -52,7 +52,8 @@ CTEST_SETUP(splinterdb_stress)
                                              .cache_size = 1000 * Mega,
                                              .disk_size  = 9000 * Mega,
                                              .data_cfg   = &data->default_data_config,
-                                             .queue_scale_percent = 100};
+                                             .num_memtable_bg_threads = 2,
+                                             .num_normal_bg_threads   = 2};
```

Comment:
I don't understand this change here.

Reply:
@ajhconway - In the old code `queue_scale_percent = 100` is a config parameter dealing with bg-threads, and its default is (already) 100%. We were not enabling bg-threads (at all) in the old code, so this line was basically redundant / unnecessary.

As the new test case has a single client loading large #s (100 M) of rows, I wanted to enable bg-threads (a) to just get the test running faster, and (b) to exercise this bg-threads machinery. We are not exercising bg-threads heavily in any large volume test. 

Thus, the introduction of above 2 lines which will run all cases in this test with 2 bg-threads of each kind enabled.

Finally, do note that this newly added test case is still failing due to the issue documented in the top-level description. So, as part of this check-in I will anyway mark this test case as `SKIP`. The code will be in there but not enabled, till we find a way to fix the `"filter->addr != 0".` assertion failure. When the test case is re-enabled, the presence of these bg-threads configs will ensure that this new test case will be exercised along with bg-threads.

Hope this clears up the reasons for this and my line of thinking.